internal/controllers/clusterextension_controller.go

```diff
@@ -184,11 +215,11 @@ func (r *ClusterExtensionReconciler) handleResolutionErrors(ext *ocv1alpha1.Clus
 	if errors.As(err, &aggErrs) {
 		for _, err := range aggErrs.Errors() {
 			errorMessage := err.Error()
-			if strings.Contains(errorMessage, "no package") {
+			if _, ok := err.(NoPackageError); ok {
```

Comment:
As the linting error says attached to this line, I think we should be using `errors.As()` just like was done above on line 215 for detecting it as the aggregated error type

Reply:
Hmmm... this shouldn't be an aggregated error type, but you are correct.

Reply:
Sorry, I was referring to how on line 215 we are using `errors.As()` to ensure the error is an `Aggregated` error. Not that this particular line should be checking for the same type.

Reply:
I knew what you meant. :)